qa/integration/services/logstash_service.rb

```diff
@@ -38,7 +38,10 @@ def initialize(settings)
       # First try without the snapshot if it's there
       @logstash_home = File.expand_path(File.join(LS_BUILD_DIR, ls_file), __FILE__)
       @logstash_home += "-SNAPSHOT" unless Dir.exists?(@logstash_home)
-
+      
+      @logstash_queue_path = File.join("#{@logstash_home}", "data")
```

Comment:
Actually, the queue path is `LS_HOME/data/queue`. I would update the variable here to `@logstash_data_path`